gh-docs/Admin_Services.md

```diff
@@ -0,0 +1,57 @@
+# How to create admin-services
+Admin Services generally follow the following steps. Here, the Message Processor has been considered as an example
+
+### Creation Flow
+
+1. Define your Admin Service function
+2. Generate the stub using WSDL
+3. Define function in MessageProcessorAdminServiceClient to call Admin Service function
+4. Call the function from the Message Processor UI jsp
+
+### 1. Defining your Admin Service
+Message Processor Admin Services are defined in the 'MessageProcessorAdminService.java' class in the carbon-mediation repo.
```

Comment:
Shall we add the link to the carbon-mediation repo?

Reply:
I am making the necessary changes in a different pull request. I will link the PR here as well once I am done.